content/docs/contributing/release-process.md

```diff
@@ -211,11 +218,11 @@ page if a step is missing or if it is outdated.
        `@maelvls`) with actual links using the following command:
 
        ```bash
-       sed github-release-description.md \
+       sed \
          -e 's$#([0-9]+)$[#\1](https://github.com/cert-manager/cert-manager/pull/\1)$g' \
          -e 's$@(\w+)$[@\1](https://github.com/\1)$g' \
          -E \
-         -i
+         github-release-description.md >release-notes.md
```

Comment:
I'm getting this error:
```
$ sed \
  -e 's$#([0-9]+)$[#\1](https://github.com/cert-manager/cert-manager/pull/\1)$g' \
  -e 's$@(\w+)$[@\1](https://github.com/\1)$g' \
  -E \
  github-release-description.md >release-notes.md
sed: -e expression #1, char 73: invalid reference \1 on `s' command's RHS
```

Reply:
Oops, `-E` should have been before `-e`. `-E` was ineffective. It should now work.